pom.xml

```diff
@@ -232,30 +232,6 @@
                     <artifactId>maven-resources-plugin</artifactId>
                     <version>${version.maven.resources}</version>
                 </plugin>
-                <plugin>
-                    <groupId>org.eclipse.m2e</groupId>
-                    <artifactId>lifecycle-mapping</artifactId>
-                    <version>1.0.0</version>
-                    <configuration>
-                        <lifecycleMappingMetadata>
-                            <pluginExecutions>
-                                <pluginExecution>
-                                    <pluginExecutionFilter>
-                                        <groupId>org.apache.maven.plugins</groupId>
-                                           <artifactId>maven-antrun-plugin</artifactId>
-                                           <versionRange>[1.7,)</versionRange>
-                                           <goals>
-                                               <goal>run</goal>
-                                           </goals>
-                                           <action>
-                                               <ignore />
-                                           </action>
-                                    </pluginExecutionFilter>
-                                </pluginExecution>
-                            </pluginExecutions>
-                        </lifecycleMappingMetadata>
-                    </configuration>
-                </plugin>
```

Comment:
@whitingjr I see you added this to overcome an issue with eclipse, but I can not see what caused the issue with eclipse.  I don't understand what this plugin is doing/why it resolves the issue you had opening the project in eclipse.  this plugin is creating issues with opening the project in vscode. 
Please can you check this PR opens in eclipse?

Reply:
I was trying to find the conversation we initially had about the problems using vscode, but have not been able to find them

Reply:
It was "live" ... we turned the end of a CPT workgroup meeting into a debugging session looking at my IDE complaints, and this is the extent of my notes on the subject: "Stäle Pedersen, John O’Hara helped me with my IDE setup. John thought the lifecycle plugin config was totally an eclipse hack (which had been implied by the google hit I found), and that it was obsolete. Indeed, both vscode and eclipse seem happy without it"

Reply:
i thought we had a discussion involving Jesper as well, but my memory might be failing me and I can't find reference to that discussion anywhere :man_shrugging:

Reply:
this is the comment in Git `Solve issue with Eclipse build triggering repeatedly.`.